### ISSUE TITLE
The reproduced results are not same as that in the paper.

### ISSUE THREAD

**Issue by imzhangyd** [HaojunYu1998/UltraDet#3]
I have used the provided pretrained model to excute test, but the obtained results (as bolow) are not same as that reported in the paper. The obtained results are higher. I'd like to learn the reasons behind this. Thanks!😃

The command is
```
python train_net.py --num-gpus 4 --eval-only  --resume \
--config-file 'configs/Base-UltraDet_R34_C4.yaml' \
DATASETS.SPLIT "test" \
DATASETS.NUM_FOLDS 1 \
MODEL.WEIGHTS 'pretrained_models/ultradet.pth' 
```

The evaluation results are
![image](https://github.com/HaojunYu1998/UltraDet/assets/39505533/8b93eda0-a04b-4da9-a5d1-ac61976093f0)

**Comment by xjtulyc** [comment]
Apologies for the confusion. The weights you have received are obtained based on the fine-tuning of hyperparameters using the validation set. The model's performance may be better than what is shown in the paper due to this tuning process.
